addon/templates/components/ember-wormhole.hbs

```diff
@@ -0,0 +1,3 @@
+{{ember-wormhole-placeholder (unbound _firstNode) ~}}
+{{yield ~}}
+{{ember-wormhole-placeholder (unbound _lastNode) ~}}
```

Comment:
@mixonic With the disclaimer that I haven't fastboot-ed, can you explain how this differs from:

```hbs
{{unbound _firstNode ~}}
{{yield ~}}
{{unbound _lastNode ~}}
```

If these approaches are functionally equivalent, we could ditch the placeholder helper (and rename `_firstNode` and `_lastNode` to something more expressive).